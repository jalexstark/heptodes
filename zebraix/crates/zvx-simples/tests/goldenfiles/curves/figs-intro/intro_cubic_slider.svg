<?xml version="1.0" encoding="UTF-8"?>
<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" width="350pt" height="350pt" viewBox="0 0 350 350">
<path fill-rule="nonzero" fill="rgb(100%, 98.039216%, 90.980392%)" fill-opacity="1" stroke-width="1" stroke-linecap="butt" stroke-linejoin="miter" stroke="rgb(0%, 0%, 0%)" stroke-opacity="1" stroke-miterlimit="10" d="M 1 349 L 1 1 L 349 1 L 349 349 Z M 1 349 "/>
<path fill="none" stroke-width="1" stroke-linecap="butt" stroke-linejoin="miter" stroke="rgb(0%, 0%, 0%)" stroke-opacity="1" stroke-miterlimit="10" d="M 15.910156 334.089844 L 15.910156 15.910156 L 334.089844 15.910156 L 334.089844 334.089844 Z M 15.910156 334.089844 "/>
<path fill="none" stroke-width="0.45" stroke-linecap="butt" stroke-linejoin="miter" stroke="rgb(0%, 0%, 0%)" stroke-opacity="1" stroke-dasharray="4.5 3.15" stroke-miterlimit="10" d="M 15.910156 175 L 334.089844 175 M 15.910156 214.773438 L 334.089844 214.773438 M 15.910156 254.546875 L 334.089844 254.546875 M 15.910156 294.316406 L 334.089844 294.316406 M 15.910156 135.226562 L 334.089844 135.226562 M 15.910156 95.453125 L 334.089844 95.453125 M 15.910156 55.683594 L 334.089844 55.683594 "/>
<path fill="none" stroke-width="0.45" stroke-linecap="butt" stroke-linejoin="miter" stroke="rgb(0%, 0%, 0%)" stroke-opacity="1" stroke-dasharray="4.5 3.15" stroke-miterlimit="10" d="M 175 334.089844 L 175 15.910156 M 135.226562 334.089844 L 135.226562 15.910156 M 95.453125 334.089844 L 95.453125 15.910156 M 55.683594 334.089844 L 55.683594 15.910156 M 214.773438 334.089844 L 214.773438 15.910156 M 254.546875 334.089844 L 254.546875 15.910156 M 294.316406 334.089844 L 294.316406 15.910156 "/>
<path fill="none" stroke-width="1" stroke-linecap="butt" stroke-linejoin="miter" stroke="rgb(45%, 18%, 0%)" stroke-opacity="1" stroke-miterlimit="10" d="M 88.3125 294.316406 C 88.3125 298.050781 82.710938 298.050781 82.710938 294.316406 C 82.710938 290.585938 88.3125 290.585938 88.3125 294.316406 Z M 267.289062 294.316406 C 267.289062 298.050781 261.6875 298.050781 261.6875 294.316406 C 261.6875 290.585938 267.289062 290.585938 267.289062 294.316406 Z M 267.289062 294.316406 "/>
<path fill="none" stroke-width="1" stroke-linecap="butt" stroke-linejoin="miter" stroke="rgb(45%, 18%, 0%)" stroke-opacity="1" stroke-miterlimit="10" d="M 109.425781 246.644531 L 115.289062 252.503906 M 115.289062 246.644531 L 109.425781 252.503906 Z M 261.558594 201.898438 L 267.417969 207.761719 M 267.417969 201.898438 L 261.558594 207.761719 Z M 267.417969 201.898438 "/>
<path fill="none" stroke-width="0.45" stroke-linecap="butt" stroke-linejoin="miter" stroke="rgb(45%, 18%, 0%)" stroke-opacity="1" stroke-dasharray="4.5 3.15" stroke-miterlimit="10" d="M 85.511719 294.316406 L 112.359375 249.574219 M 264.488281 294.316406 L 264.488281 204.828125 "/>
<path fill="none" stroke-width="1" stroke-linecap="butt" stroke-linejoin="miter" stroke="rgb(45%, 18%, 0%)" stroke-opacity="1" stroke-miterlimit="10" d="M 303.085938 294.316406 C 303.085938 298.050781 297.484375 298.050781 297.484375 294.316406 C 297.484375 290.585938 303.085938 290.585938 303.085938 294.316406 Z M 303.085938 25.851562 C 303.085938 29.585938 297.484375 29.585938 297.484375 25.851562 C 297.484375 22.117188 303.085938 22.117188 303.085938 25.851562 Z M 303.085938 25.851562 "/>
<path fill="none" stroke-width="1" stroke-linecap="butt" stroke-linejoin="miter" stroke="rgb(45%, 18%, 0%)" stroke-opacity="1" stroke-miterlimit="10" d="M 297.355469 201.898438 L 303.214844 207.761719 M 303.214844 201.898438 L 297.355469 207.761719 Z M 297.355469 112.410156 L 303.214844 118.269531 M 303.214844 112.410156 L 297.355469 118.269531 Z M 303.214844 112.410156 "/>
<path fill="none" stroke-width="0.45" stroke-linecap="butt" stroke-linejoin="miter" stroke="rgb(45%, 18%, 0%)" stroke-opacity="1" stroke-dasharray="4.5 3.15" stroke-miterlimit="10" d="M 300.285156 294.316406 L 300.285156 204.828125 M 300.285156 25.851562 L 300.285156 115.339844 "/>
<path fill="none" stroke-width="1" stroke-linecap="butt" stroke-linejoin="miter" stroke="rgb(45%, 18%, 0%)" stroke-opacity="1" stroke-miterlimit="10" d="M 57.617188 275.902344 C 57.617188 279.636719 52.015625 279.636719 52.015625 275.902344 C 52.015625 272.167969 57.617188 272.167969 57.617188 275.902344 Z M 138.15625 141.667969 C 138.15625 145.402344 132.558594 145.402344 132.558594 141.667969 C 132.558594 137.933594 138.15625 137.933594 138.15625 141.667969 Z M 138.15625 141.667969 "/>
<path fill="none" stroke-width="1" stroke-linecap="butt" stroke-linejoin="miter" stroke="rgb(45%, 18%, 0%)" stroke-opacity="1" stroke-miterlimit="10" d="M 78.734375 228.226562 L 84.59375 234.085938 M 84.59375 228.226562 L 78.734375 234.085938 Z M 105.578125 183.484375 L 111.441406 189.34375 M 111.441406 183.484375 L 105.578125 189.34375 Z M 111.441406 183.484375 "/>
<path fill="none" stroke-width="0.45" stroke-linecap="butt" stroke-linejoin="miter" stroke="rgb(45%, 18%, 0%)" stroke-opacity="1" stroke-dasharray="4.5 3.15" stroke-miterlimit="10" d="M 54.816406 275.902344 L 81.664062 231.15625 M 135.355469 141.667969 L 108.511719 186.414062 "/>
<path fill="none" stroke-width="1" stroke-linecap="butt" stroke-linejoin="miter" stroke="rgb(45%, 18%, 0%)" stroke-opacity="1" stroke-miterlimit="10" d="M 58.375 221.34375 C 58.375 225.078125 52.777344 225.078125 52.777344 221.34375 C 52.777344 217.613281 58.375 217.613281 58.375 221.34375 Z M 306.707031 187.789062 C 306.707031 191.519531 301.105469 191.519531 301.105469 187.789062 C 301.105469 184.054688 306.707031 184.054688 306.707031 187.789062 Z M 306.707031 187.789062 "/>
<path fill="none" stroke-width="1" stroke-linecap="butt" stroke-linejoin="miter" stroke="rgb(45%, 18%, 0%)" stroke-opacity="1" stroke-miterlimit="10" d="M 135.421875 207.230469 L 141.28125 213.089844 M 141.28125 207.230469 L 135.421875 213.089844 Z M 218.199219 196.042969 L 224.058594 201.902344 M 224.058594 196.042969 L 218.199219 201.902344 Z M 224.058594 196.042969 "/>
<path fill="none" stroke-width="0.45" stroke-linecap="butt" stroke-linejoin="miter" stroke="rgb(45%, 18%, 0%)" stroke-opacity="1" stroke-dasharray="4.5 3.15" stroke-miterlimit="10" d="M 55.574219 221.34375 L 138.351562 210.160156 M 303.90625 187.789062 L 221.128906 198.972656 "/>
<path fill-rule="nonzero" fill="rgb(0%, 0%, 65%)" fill-opacity="1" d="M 97.25 283.210938 C 97.25 286.644531 92.097656 286.644531 92.097656 283.210938 C 92.097656 279.773438 97.25 279.773438 97.25 283.210938 "/>
<path fill-rule="nonzero" fill="rgb(0%, 0%, 65%)" fill-opacity="1" d="M 110.667969 272.566406 C 110.667969 276.003906 105.515625 276.003906 105.515625 272.566406 C 105.515625 269.132812 110.667969 269.132812 110.667969 272.566406 "/>
<path fill-rule="nonzero" fill="rgb(0%, 0%, 65%)" fill-opacity="1" d="M 127.378906 262.855469 C 127.378906 266.292969 122.226562 266.292969 122.226562 262.855469 C 122.226562 259.421875 127.378906 259.421875 127.378906 262.855469 "/>
<path fill-rule="nonzero" fill="rgb(0%, 0%, 65%)" fill-opacity="1" d="M 146.421875 254.546875 C 146.421875 257.980469 141.269531 257.980469 141.269531 254.546875 C 141.269531 251.109375 146.421875 251.109375 146.421875 254.546875 "/>
<path fill-rule="nonzero" fill="rgb(0%, 0%, 65%)" fill-opacity="1" d="M 166.832031 248.097656 C 166.832031 251.53125 161.679688 251.53125 161.679688 248.097656 C 161.679688 244.664062 166.832031 244.664062 166.832031 248.097656 "/>
<path fill-rule="nonzero" fill="rgb(0%, 0%, 65%)" fill-opacity="1" d="M 187.644531 243.980469 C 187.644531 247.414062 182.492188 247.414062 182.492188 243.980469 C 182.492188 240.546875 187.644531 240.546875 187.644531 243.980469 "/>
<path fill-rule="nonzero" fill="rgb(0%, 0%, 65%)" fill-opacity="1" d="M 207.898438 242.660156 C 207.898438 246.09375 202.746094 246.09375 202.746094 242.660156 C 202.746094 239.226562 207.898438 239.226562 207.898438 242.660156 "/>
<path fill-rule="nonzero" fill="rgb(0%, 0%, 65%)" fill-opacity="1" d="M 226.628906 244.601562 C 226.628906 248.035156 221.476562 248.035156 221.476562 244.601562 C 221.476562 241.167969 226.628906 241.167969 226.628906 244.601562 "/>
<path fill-rule="nonzero" fill="rgb(0%, 0%, 65%)" fill-opacity="1" d="M 242.875 250.273438 C 242.875 253.707031 237.722656 253.707031 237.722656 250.273438 C 237.722656 246.839844 242.875 246.839844 242.875 250.273438 "/>
<path fill-rule="nonzero" fill="rgb(0%, 0%, 65%)" fill-opacity="1" d="M 255.671875 260.136719 C 255.671875 263.574219 250.519531 263.574219 250.519531 260.136719 C 250.519531 256.703125 255.671875 256.703125 255.671875 260.136719 "/>
<path fill-rule="nonzero" fill="rgb(0%, 0%, 65%)" fill-opacity="1" d="M 264.054688 274.664062 C 264.054688 278.097656 258.902344 278.097656 258.902344 274.664062 C 258.902344 271.230469 264.054688 271.230469 264.054688 274.664062 "/>
<path fill-rule="nonzero" fill="rgb(0%, 0%, 65%)" fill-opacity="1" d="M 302.859375 271.945312 C 302.859375 275.378906 297.707031 275.378906 297.707031 271.945312 C 297.707031 268.511719 302.859375 268.511719 302.859375 271.945312 "/>
<path fill-rule="nonzero" fill="rgb(0%, 0%, 65%)" fill-opacity="1" d="M 302.859375 249.574219 C 302.859375 253.007812 297.707031 253.007812 297.707031 249.574219 C 297.707031 246.140625 302.859375 246.140625 302.859375 249.574219 "/>
<path fill-rule="nonzero" fill="rgb(0%, 0%, 65%)" fill-opacity="1" d="M 302.859375 227.203125 C 302.859375 230.636719 297.707031 230.636719 297.707031 227.203125 C 297.707031 223.765625 302.859375 223.765625 302.859375 227.203125 "/>
<path fill-rule="nonzero" fill="rgb(0%, 0%, 65%)" fill-opacity="1" d="M 302.859375 204.828125 C 302.859375 208.265625 297.707031 208.265625 297.707031 204.828125 C 297.707031 201.394531 302.859375 201.394531 302.859375 204.828125 "/>
<path fill-rule="nonzero" fill="rgb(0%, 0%, 65%)" fill-opacity="1" d="M 302.859375 182.457031 C 302.859375 185.890625 297.707031 185.890625 297.707031 182.457031 C 297.707031 179.023438 302.859375 179.023438 302.859375 182.457031 "/>
<path fill-rule="nonzero" fill="rgb(0%, 0%, 65%)" fill-opacity="1" d="M 302.859375 160.085938 C 302.859375 163.519531 297.707031 163.519531 297.707031 160.085938 C 297.707031 156.652344 302.859375 156.652344 302.859375 160.085938 "/>
<path fill-rule="nonzero" fill="rgb(0%, 0%, 65%)" fill-opacity="1" d="M 302.859375 137.714844 C 302.859375 141.148438 297.707031 141.148438 297.707031 137.714844 C 297.707031 134.277344 302.859375 134.277344 302.859375 137.714844 "/>
<path fill-rule="nonzero" fill="rgb(0%, 0%, 65%)" fill-opacity="1" d="M 302.859375 115.339844 C 302.859375 118.777344 297.707031 118.777344 297.707031 115.339844 C 297.707031 111.90625 302.859375 111.90625 302.859375 115.339844 "/>
<path fill-rule="nonzero" fill="rgb(0%, 0%, 65%)" fill-opacity="1" d="M 302.859375 92.96875 C 302.859375 96.402344 297.707031 96.402344 297.707031 92.96875 C 297.707031 89.535156 302.859375 89.535156 302.859375 92.96875 "/>
<path fill-rule="nonzero" fill="rgb(0%, 0%, 65%)" fill-opacity="1" d="M 302.859375 70.597656 C 302.859375 74.03125 297.707031 74.03125 297.707031 70.597656 C 297.707031 67.160156 302.859375 67.160156 302.859375 70.597656 "/>
<path fill-rule="nonzero" fill="rgb(0%, 0%, 65%)" fill-opacity="1" d="M 302.859375 48.222656 C 302.859375 51.660156 297.707031 51.660156 297.707031 48.222656 C 297.707031 44.789062 302.859375 44.789062 302.859375 48.222656 "/>
<path fill-rule="nonzero" fill="rgb(0%, 0%, 65%)" fill-opacity="1" d="M 64.105469 264.714844 C 64.105469 268.148438 58.953125 268.148438 58.953125 264.714844 C 58.953125 261.28125 64.105469 261.28125 64.105469 264.714844 "/>
<path fill-rule="nonzero" fill="rgb(0%, 0%, 65%)" fill-opacity="1" d="M 70.816406 253.53125 C 70.816406 256.964844 65.664062 256.964844 65.664062 253.53125 C 65.664062 250.09375 70.816406 250.09375 70.816406 253.53125 "/>
<path fill-rule="nonzero" fill="rgb(0%, 0%, 65%)" fill-opacity="1" d="M 77.527344 242.34375 C 77.527344 245.777344 72.375 245.777344 72.375 242.34375 C 72.375 238.910156 77.527344 238.910156 77.527344 242.34375 "/>
<path fill-rule="nonzero" fill="rgb(0%, 0%, 65%)" fill-opacity="1" d="M 84.238281 231.15625 C 84.238281 234.59375 79.085938 234.59375 79.085938 231.15625 C 79.085938 227.722656 84.238281 227.722656 84.238281 231.15625 "/>
<path fill-rule="nonzero" fill="rgb(0%, 0%, 65%)" fill-opacity="1" d="M 90.953125 219.972656 C 90.953125 223.40625 85.800781 223.40625 85.800781 219.972656 C 85.800781 216.535156 90.953125 216.535156 90.953125 219.972656 "/>
<path fill-rule="nonzero" fill="rgb(0%, 0%, 65%)" fill-opacity="1" d="M 97.664062 208.785156 C 97.664062 212.21875 92.511719 212.21875 92.511719 208.785156 C 92.511719 205.351562 97.664062 205.351562 97.664062 208.785156 "/>
<path fill-rule="nonzero" fill="rgb(0%, 0%, 65%)" fill-opacity="1" d="M 104.375 197.597656 C 104.375 201.035156 99.222656 201.035156 99.222656 197.597656 C 99.222656 194.164062 104.375 194.164062 104.375 197.597656 "/>
<path fill-rule="nonzero" fill="rgb(0%, 0%, 65%)" fill-opacity="1" d="M 111.085938 186.414062 C 111.085938 189.847656 105.933594 189.847656 105.933594 186.414062 C 105.933594 182.976562 111.085938 182.976562 111.085938 186.414062 "/>
<path fill-rule="nonzero" fill="rgb(0%, 0%, 65%)" fill-opacity="1" d="M 117.796875 175.226562 C 117.796875 178.660156 112.644531 178.660156 112.644531 175.226562 C 112.644531 171.792969 117.796875 171.792969 117.796875 175.226562 "/>
<path fill-rule="nonzero" fill="rgb(0%, 0%, 65%)" fill-opacity="1" d="M 124.507812 164.039062 C 124.507812 167.476562 119.359375 167.476562 119.359375 164.039062 C 119.359375 160.605469 124.507812 160.605469 124.507812 164.039062 "/>
<path fill-rule="nonzero" fill="rgb(0%, 0%, 65%)" fill-opacity="1" d="M 131.222656 152.855469 C 131.222656 156.289062 126.070312 156.289062 126.070312 152.855469 C 126.070312 149.421875 131.222656 149.421875 131.222656 152.855469 "/>
<path fill-rule="nonzero" fill="rgb(0%, 0%, 65%)" fill-opacity="1" d="M 78.847656 218.550781 C 78.847656 221.984375 73.695312 221.984375 73.695312 218.550781 C 73.695312 215.113281 78.847656 215.113281 78.847656 218.550781 "/>
<path fill-rule="nonzero" fill="rgb(0%, 0%, 65%)" fill-opacity="1" d="M 99.539062 215.753906 C 99.539062 219.1875 94.386719 219.1875 94.386719 215.753906 C 94.386719 212.316406 99.539062 212.316406 99.539062 215.753906 "/>
<path fill-rule="nonzero" fill="rgb(0%, 0%, 65%)" fill-opacity="1" d="M 120.234375 212.957031 C 120.234375 216.390625 115.082031 216.390625 115.082031 212.957031 C 115.082031 209.519531 120.234375 209.519531 120.234375 212.957031 "/>
<path fill-rule="nonzero" fill="rgb(0%, 0%, 65%)" fill-opacity="1" d="M 140.929688 210.160156 C 140.929688 213.59375 135.777344 213.59375 135.777344 210.160156 C 135.777344 206.726562 140.929688 206.726562 140.929688 210.160156 "/>
<path fill-rule="nonzero" fill="rgb(0%, 0%, 65%)" fill-opacity="1" d="M 161.621094 207.363281 C 161.621094 210.796875 156.472656 210.796875 156.472656 207.363281 C 156.472656 203.929688 161.621094 203.929688 161.621094 207.363281 "/>
<path fill-rule="nonzero" fill="rgb(0%, 0%, 65%)" fill-opacity="1" d="M 182.316406 204.566406 C 182.316406 208 177.164062 208 177.164062 204.566406 C 177.164062 201.132812 182.316406 201.132812 182.316406 204.566406 "/>
<path fill-rule="nonzero" fill="rgb(0%, 0%, 65%)" fill-opacity="1" d="M 203.011719 201.769531 C 203.011719 205.203125 197.859375 205.203125 197.859375 201.769531 C 197.859375 198.335938 203.011719 198.335938 203.011719 201.769531 "/>
<path fill-rule="nonzero" fill="rgb(0%, 0%, 65%)" fill-opacity="1" d="M 223.707031 198.972656 C 223.707031 202.40625 218.554688 202.40625 218.554688 198.972656 C 218.554688 195.539062 223.707031 195.539062 223.707031 198.972656 "/>
<path fill-rule="nonzero" fill="rgb(0%, 0%, 65%)" fill-opacity="1" d="M 244.398438 196.175781 C 244.398438 199.613281 239.246094 199.613281 239.246094 196.175781 C 239.246094 192.742188 244.398438 192.742188 244.398438 196.175781 "/>
<path fill-rule="nonzero" fill="rgb(0%, 0%, 65%)" fill-opacity="1" d="M 265.09375 193.378906 C 265.09375 196.816406 259.941406 196.816406 259.941406 193.378906 C 259.941406 189.945312 265.09375 189.945312 265.09375 193.378906 "/>
<path fill-rule="nonzero" fill="rgb(0%, 0%, 65%)" fill-opacity="1" d="M 285.789062 190.582031 C 285.789062 194.019531 280.636719 194.019531 280.636719 190.582031 C 280.636719 187.148438 285.789062 187.148438 285.789062 190.582031 "/>
<path fill="none" stroke-width="1" stroke-linecap="butt" stroke-linejoin="miter" stroke="rgb(0%, 40%, 0%)" stroke-opacity="1" stroke-miterlimit="10" d="M 85.511719 294.316406 C 112.359375 249.574219 264.488281 204.828125 264.488281 294.316406 "/>
<path fill="none" stroke-width="1" stroke-linecap="butt" stroke-linejoin="miter" stroke="rgb(0%, 40%, 0%)" stroke-opacity="1" stroke-miterlimit="10" d="M 300.285156 294.316406 C 300.285156 204.828125 300.285156 115.339844 300.285156 25.851562 "/>
<path fill="none" stroke-width="1" stroke-linecap="butt" stroke-linejoin="miter" stroke="rgb(0%, 40%, 0%)" stroke-opacity="1" stroke-miterlimit="10" d="M 54.816406 275.902344 C 81.664062 231.15625 108.511719 186.414062 135.355469 141.667969 "/>
<path fill="none" stroke-width="1" stroke-linecap="butt" stroke-linejoin="miter" stroke="rgb(0%, 40%, 0%)" stroke-opacity="1" stroke-miterlimit="10" d="M 55.574219 221.34375 C 138.351562 210.160156 221.128906 198.972656 303.90625 187.789062 "/>
</svg>
