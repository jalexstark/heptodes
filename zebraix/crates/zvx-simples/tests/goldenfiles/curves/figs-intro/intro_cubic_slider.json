{
  "drawables": [
    {
      "layer": -10,
      "drawable": {
        "SegmentSequence": {
          "completion": "Closed",
          "path_choices": {
            "fill_choices": {
              "color": "ZvxBackground",
              "opacity": 1.0
            }
          },
          "segments": [
            {
              "Polyline": [
                [
                  -4.374857142857143,
                  -4.374857142857143
                ],
                [
                  -4.374857142857143,
                  4.374857142857143
                ],
                [
                  4.374857142857143,
                  4.374857142857143
                ],
                [
                  4.374857142857143,
                  -4.374857142857143
                ]
              ]
            }
          ]
        }
      }
    },
    {
      "drawable": {
        "SegmentSequence": {
          "completion": "Closed",
          "segments": [
            {
              "Polyline": [
                [
                  -4.0,
                  -4.0
                ],
                [
                  -4.0,
                  4.0
                ],
                [
                  4.0,
                  4.0
                ],
                [
                  4.0,
                  -4.0
                ]
              ]
            }
          ]
        }
      }
    },
    {
      "drawable": {
        "Lines": {
          "path": {
            "coords": [
              [
                [
                  -4.0,
                  0.0
                ],
                [
                  4.0,
                  0.0
                ]
              ]
            ],
            "offsets": [
              [
                0.0,
                0.0
              ],
              [
                -0.0,
                -1.0
              ],
              [
                -0.0,
                -2.0
              ],
              [
                -0.0,
                -3.0
              ],
              [
                0.0,
                1.0
              ],
              [
                0.0,
                2.0
              ],
              [
                0.0,
                3.0
              ]
            ]
          },
          "path_choices": {
            "line_choice": "Light"
          }
        }
      }
    },
    {
      "drawable": {
        "Lines": {
          "path": {
            "coords": [
              [
                [
                  0.0,
                  -4.0
                ],
                [
                  0.0,
                  4.0
                ]
              ]
            ],
            "offsets": [
              [
                0.0,
                0.0
              ],
              [
                -1.0,
                -0.0
              ],
              [
                -2.0,
                -0.0
              ],
              [
                -3.0,
                -0.0
              ],
              [
                1.0,
                0.0
              ],
              [
                2.0,
                0.0
              ],
              [
                3.0,
                0.0
              ]
            ]
          },
          "path_choices": {
            "line_choice": "Light"
          }
        }
      }
    },
    {
      "layer": 10,
      "drawable": {
        "Points": {
          "color_choice": "RedRedGreen",
          "centers": [
            [
              -2.25,
              -3.0
            ],
            [
              2.25,
              -3.0
            ]
          ]
        }
      }
    },
    {
      "layer": 10,
      "drawable": {
        "Points": {
          "point_choice": "Times",
          "color_choice": "RedRedGreen",
          "centers": [
            [
              -1.575,
              -1.875
            ],
            [
              2.25,
              -0.75
            ]
          ]
        }
      }
    },
    {
      "layer": 10,
      "drawable": {
        "Lines": {
          "path": {
            "coords": [
              [
                [
                  -2.25,
                  -3.0
                ],
                [
                  -1.575,
                  -1.875
                ]
              ],
              [
                [
                  2.25,
                  -3.0
                ],
                [
                  2.25,
                  -0.75
                ]
              ]
            ]
          },
          "path_choices": {
            "line_choice": "Light",
            "color": "RedRedGreen"
          }
        }
      }
    },
    {
      "layer": 20,
      "drawable": {
        "Points": {
          "point_choice": "Dot",
          "color_choice": "Blue",
          "centers": [
            [
              -2.0196614583333337,
              -2.7207031250000004
            ],
            [
              -1.6822916666666672,
              -2.4531250000000004
            ],
            [
              -1.2621093750000003,
              -2.2089843750000004
            ],
            [
              -0.7833333333333331,
              -2.0
            ],
            [
              -0.2701822916666665,
              -1.8378906250000002
            ],
            [
              0.25312500000000004,
              -1.7343750000000004
            ],
            [
              0.7623697916666672,
              -1.7011718750000004
            ],
            [
              1.233333333333334,
              -1.7500000000000007
            ],
            [
              1.6417968750000005,
              -1.8925781250000004
            ],
            [
              1.9635416666666672,
              -2.1406250000000004
            ],
            [
              2.1743489583333346,
              -2.505859375000001
            ]
          ]
        }
      }
    },
    {
      "layer": 30,
      "drawable": {
        "Cubic": {
          "path": {
            "r": [
              -6.0,
              14.0
            ],
            "p": [
              [
                -2.25,
                -3.0
              ],
              [
                -1.575,
                -1.875
              ],
              [
                2.25,
                -0.75
              ],
              [
                2.25,
                -3.0
              ]
            ]
          },
          "path_choices": {
            "color": "Green"
          }
        }
      }
    },
    {
      "layer": 10,
      "drawable": {
        "Points": {
          "color_choice": "RedRedGreen",
          "centers": [
            [
              3.15,
              -3.0
            ],
            [
              3.15,
              3.75
            ]
          ]
        }
      }
    },
    {
      "layer": 10,
      "drawable": {
        "Points": {
          "point_choice": "Times",
          "color_choice": "RedRedGreen",
          "centers": [
            [
              3.15,
              -0.75
            ],
            [
              3.15,
              1.5
            ]
          ]
        }
      }
    },
    {
      "layer": 10,
      "drawable": {
        "Lines": {
          "path": {
            "coords": [
              [
                [
                  3.15,
                  -3.0
                ],
                [
                  3.15,
                  -0.75
                ]
              ],
              [
                [
                  3.15,
                  3.75
                ],
                [
                  3.15,
                  1.5
                ]
              ]
            ]
          },
          "path_choices": {
            "line_choice": "Light",
            "color": "RedRedGreen"
          }
        }
      }
    },
    {
      "layer": 20,
      "drawable": {
        "Points": {
          "point_choice": "Dot",
          "color_choice": "Blue",
          "centers": [
            [
              3.1500000000000004,
              -2.4375000000000004
            ],
            [
              3.1500000000000012,
              -1.8750000000000007
            ],
            [
              3.150000000000001,
              -1.3125000000000002
            ],
            [
              3.1500000000000004,
              -0.7499999999999999
            ],
            [
              3.1500000000000012,
              -0.18749999999999975
            ],
            [
              3.150000000000001,
              0.3750000000000001
            ],
            [
              3.150000000000001,
              0.9375000000000007
            ],
            [
              3.1500000000000012,
              1.5000000000000009
            ],
            [
              3.150000000000001,
              2.0625000000000004
            ],
            [
              3.150000000000001,
              2.6250000000000004
            ],
            [
              3.1500000000000012,
              3.1875000000000018
            ]
          ]
        }
      }
    },
    {
      "layer": 30,
      "drawable": {
        "Cubic": {
          "path": {
            "r": [
              -6.0,
              14.0
            ],
            "p": [
              [
                3.15,
                -3.0
              ],
              [
                3.15,
                -0.75
              ],
              [
                3.15,
                1.5
              ],
              [
                3.15,
                3.75
              ]
            ]
          },
          "path_choices": {
            "color": "Green"
          }
        }
      }
    },
    {
      "layer": 10,
      "drawable": {
        "Points": {
          "color_choice": "RedRedGreen",
          "centers": [
            [
              -3.0217436331412895,
              -2.536953820115226
            ],
            [
              -0.9967436331412891,
              0.838046179884774
            ]
          ]
        }
      }
    },
    {
      "layer": 10,
      "drawable": {
        "Points": {
          "point_choice": "Times",
          "color_choice": "RedRedGreen",
          "centers": [
            [
              -2.3467436331412896,
              -1.411953820115226
            ],
            [
              -1.6717436331412894,
              -0.28695382011522597
            ]
          ]
        }
      }
    },
    {
      "layer": 10,
      "drawable": {
        "Lines": {
          "path": {
            "coords": [
              [
                [
                  -3.0217436331412895,
                  -2.536953820115226
                ],
                [
                  -2.3467436331412896,
                  -1.411953820115226
                ]
              ],
              [
                [
                  -0.9967436331412891,
                  0.838046179884774
                ],
                [
                  -1.6717436331412894,
                  -0.28695382011522597
                ]
              ]
            ]
          },
          "path_choices": {
            "line_choice": "Light",
            "color": "RedRedGreen"
          }
        }
      }
    },
    {
      "layer": 20,
      "drawable": {
        "Points": {
          "point_choice": "Dot",
          "color_choice": "Blue",
          "centers": [
            [
              -2.8529936331412897,
              -2.2557038201152264
            ],
            [
              -2.684243633141291,
              -1.9744538201152269
            ],
            [
              -2.51549363314129,
              -1.6932038201152262
            ],
            [
              -2.3467436331412896,
              -1.4119538201152262
            ],
            [
              -2.1779936331412895,
              -1.1307038201152262
            ],
            [
              -2.00924363314129,
              -0.8494538201152262
            ],
            [
              -1.8404936331412898,
              -0.568203820115226
            ],
            [
              -1.67174363314129,
              -0.2869538201152259
            ],
            [
              -1.5029936331412896,
              -0.0057038201152259955
            ],
            [
              -1.3342436331412895,
              0.27554617988477426
            ],
            [
              -1.1654936331412897,
              0.5567961798847745
            ]
          ]
        }
      }
    },
    {
      "layer": 30,
      "drawable": {
        "Cubic": {
          "path": {
            "r": [
              -6.0,
              14.0
            ],
            "p": [
              [
                -3.0217436331412895,
                -2.536953820115226
              ],
              [
                -2.3467436331412896,
                -1.411953820115226
              ],
              [
                -1.6717436331412894,
                -0.28695382011522597
              ],
              [
                -0.9967436331412891,
                0.838046179884774
              ]
            ]
          },
          "path_choices": {
            "color": "Green"
          }
        }
      }
    },
    {
      "layer": 10,
      "drawable": {
        "Points": {
          "color_choice": "RedRedGreen",
          "centers": [
            [
              -3.002667896001395,
              -1.1652575695896767
            ],
            [
              3.2410821039986053,
              -0.32150756958967674
            ]
          ]
        }
      }
    },
    {
      "layer": 10,
      "drawable": {
        "Points": {
          "point_choice": "Times",
          "color_choice": "RedRedGreen",
          "centers": [
            [
              -0.9214178960013952,
              -0.8840075695896767
            ],
            [
              1.159832103998605,
              -0.6027575695896767
            ]
          ]
        }
      }
    },
    {
      "layer": 10,
      "drawable": {
        "Lines": {
          "path": {
            "coords": [
              [
                [
                  -3.002667896001395,
                  -1.1652575695896767
                ],
                [
                  -0.9214178960013952,
                  -0.8840075695896767
                ]
              ],
              [
                [
                  3.2410821039986053,
                  -0.32150756958967674
                ],
                [
                  1.159832103998605,
                  -0.6027575695896767
                ]
              ]
            ]
          },
          "path_choices": {
            "line_choice": "Light",
            "color": "RedRedGreen"
          }
        }
      }
    },
    {
      "layer": 20,
      "drawable": {
        "Points": {
          "point_choice": "Dot",
          "color_choice": "Blue",
          "centers": [
            [
              -2.4823553960013958,
              -1.094945069589677
            ],
            [
              -1.9620428960013958,
              -1.0246325695896772
            ],
            [
              -1.4417303960013952,
              -0.954320069589677
            ],
            [
              -0.9214178960013949,
              -0.8840075695896767
            ],
            [
              -0.4011053960013949,
              -0.813695069589677
            ],
            [
              0.11920710399860505,
              -0.7433825695896769
            ],
            [
              0.6395196039986055,
              -0.673070069589677
            ],
            [
              1.1598321039986057,
              -0.602757569589677
            ],
            [
              1.6801446039986052,
              -0.532445069589677
            ],
            [
              2.2004571039986063,
              -0.46213256958967686
            ],
            [
              2.7207696039986065,
              -0.3918200695896769
            ]
          ]
        }
      }
    },
    {
      "layer": 30,
      "drawable": {
        "Cubic": {
          "path": {
            "r": [
              -6.0,
              14.0
            ],
            "p": [
              [
                -3.002667896001395,
                -1.1652575695896767
              ],
              [
                -0.9214178960013952,
                -0.8840075695896767
              ],
              [
                1.159832103998605,
                -0.6027575695896767
              ],
              [
                3.2410821039986053,
                -0.32150756958967674
              ]
            ]
          },
          "path_choices": {
            "color": "Green"
          }
        }
      }
    }
  ]
}
